### DISCUSSION TITLE
what is ndarray netlib

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [rust-ndarray/ndarray#1449] [Q&A]
Hi,
I am a beginner of ndarray, who can tell what `netlib` is ?  Does it make `ndarray` faster ?

**Comment by nilgoyette** [comment]
`netlib` is one of the possible [implementations](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms#Implementations) of "BLAS". Using `netlib` or any other **could** make your code faster, but this is not always true. It depends on your data and what you do with it.

I suggest you use ndarray "normally" (without BLAS). When your project is finished, you can test the performance with and without it.
